buildspecs/update-master-from-release.yml

```diff
@@ -43,4 +45,5 @@ phases:
         git commit -am "Update to next snapshot version: $NEW_VERSION_SNAPSHOT"
       fi
     -
+    - git status
```

Comment:
Why do we need this?

Reply:
just for logging, it was present in the [other buildspec](https://github.com/aws/aws-sdk-java-v2/blob/master/buildspecs/update-snapshot-version.yml#L28)
wdyt?

Reply:
Got it, that's fine. I was curious